web/sidebar.js

```diff
@@ -48,46 +55,66 @@ class Sidebar {
   constructor({ sidebar, resizer, toggleButton }, ltr, isResizerOnTheLeft) {
     this._sidebar = sidebar;
     this.#coefficient = ltr === isResizerOnTheLeft ? -1 : 1;
+    this.#resizer = resizer;
+    this.#isResizerOnTheLeft = isResizerOnTheLeft;
 
     const style = window.getComputedStyle(sidebar);
-    this.#minWidth = parseFloat(style.getPropertyValue("--sidebar-min-width"));
-    this.#maxWidth = parseFloat(style.getPropertyValue("--sidebar-max-width"));
     this.#initialWidth = this.#width = parseFloat(
       style.getPropertyValue("--sidebar-width")
     );
+    resizer.ariaValueMin = parseFloat(
+      style.getPropertyValue("--sidebar-min-width")
+    );
+    resizer.ariaValueMax = parseFloat(
+      style.getPropertyValue("--sidebar-max-width")
+    );
+    resizer.ariaValueNow = this.#width;
 
-    this.#makeSidebarResizable(resizer, isResizerOnTheLeft);
+    this.#makeSidebarResizable();
     toggleButton.addEventListener("click", this.toggle.bind(this));
-    sidebar.hidden = true;
+    this._isOpen = sidebar.hidden = true;
```

Comment:
This seems a bit contradictory to me because if the sidebar element is hidden then I would assume `this._isOpen` to equal `false` instead since it's not visible/open?